eduvpn/crypto.py

```diff
@@ -14,61 +14,37 @@
 
 
 def common_name_from_cert(pem_data):
-    """
-    Extract common name from client certificate.
-
-    args:
-        pem_data (str): PEM encoded certificate
-
-    returns:
-        str: the common name of the client certificate.
-
-    """
-    cert = x509.load_pem_x509_certificate(pem_data, default_backend())
-    return cert.subject.get_attributes_for_oid(NameOID.COMMON_NAME)[0].value
+	#type: (str) -> str
+	"""Extract common name from client certificate."""
+	cert = x509.load_pem_x509_certificate(pem_data, default_backend())
+	return cert.subject.get_attributes_for_oid(NameOID.COMMON_NAME)[0].value
 
 
 def gen_code_verifier(length=128):
-    """
-    Generate a high entropy code verifier, used for PKCE
-
-    args:
-        length (int): length of the code
-
-    returns:
-        str:
-    """
+	#type: (int) -> str
+    """Generate a high entropy code verifier, used for PKCE."""
```

Comment:
now the arguments don't have a doc string anymore. You can leave the doc and just remove the types.